examples/gae-autoinstrument-example/README.md

```diff
@@ -0,0 +1,63 @@
+Google AppEngine Standard with OTel Auto-instrumentation
+============================
+**This sample app is a Maven project and builds separately from the rest of the rest of the examples.**
+
+This example Java app runs on GAE standard environment and is instrumented using OpenTelemetry's [auto-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation) agent.
```

Comment:
How much of this sample is really specific to AppEngine? Why couldn't one of our other samples be deployed to AppEngine?